onemask/ArchitectureStudy/app/src/main/java/com/android/studyfork/ui/adapter/ViewPagerAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package com.android.studyfork.ui.adapter
+
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentPagerAdapter
+import com.android.studyfork.ui.TickerListFragment
+
+/**
+ * created by onemask
+ */
+class ViewpagerAdapter(fragmentManager : FragmentManager) :
```

Comment:
카멜케이스로 해주는게 나을 것 같습니다
```suggestion
class ViewPagerAdapter(fragmentManager : FragmentManager) :
```

Reply:
수정하였습니다.
969708b